mucommander-core/src/main/java/com/mucommander/ui/dialog/pref/general/ShortcutsTable.java

```diff
@@ -742,10 +785,18 @@ public void submitChanges() {
             }
         }
 
-        public String contains(KeyStroke accelerator) {
+        /**
+         * Checks if a given accelerator is already present in edited action map in defined scope.
+         *
+         * @param accelerator the accelerator, must not be null
+         * @param scope a set ActionType values defining the scope, must not be null
+         * @return the ActionId, or null if not present
+         */
+        public ActionId contains(KeyStroke accelerator, Set<ActionId.ActionType> scope) {
             if (accelerator != null) {
-                for (Map.Entry<String, Map<TableDataColumnEnum, Object>> entry : editedActionMap.entrySet()) {
-                    if (accelerator.equals(entry.getValue().get(TableDataColumnEnum.ACCELERATOR)) ||
+                for (Map.Entry<ActionId, Map<TableDataColumnEnum, Object>> entry : editedActionMap.entrySet()) {
+                    if (scope.contains(entry.getKey().getType()) &&
+                            accelerator.equals(entry.getValue().get(TableDataColumnEnum.ACCELERATOR)) ||
                             accelerator.equals(entry.getValue().get(TableDataColumnEnum.ALT_ACCELERATOR))) {
```

Comment:
shouldn't this be 
scope.contains(entry.getKey().getType()) &&
                           (accelerator.equals(entry.getValue().get(TableDataColumnEnum.ACCELERATOR)) ||
                            accelerator.equals(entry.getValue().get(TableDataColumnEnum.ALT_ACCELERATOR))

Reply:
good catch